README.md

```diff
@@ -73,10 +73,10 @@ conda create -n cadquery
 conda activate cadquery
```

Comment:
Still few conda instances missed. Also l64 does mention conda

Reply:
Do you mean the instances such as `conda activate`?  I replaced only `conda install` with `mamba install` because when I began using mamba, the mamba docs recommended to continue to use `conda activate` and `conda deactivate`.  Not all conda commands had been implemented in mamba. 

The mamba docs have been updated to remove the warning (it had said you should still use conda for activation/deactivation): 
https://github.com/mamba-org/mamba/commit/63e75d8bc27b2ca35c4c602055ee4dfa57ef2059#diff-863a289903793c21e000164624119574147af60f75ecbc04ca3d511779c10798L54

Though the warning is removed my understanding is that `mamba activate` is not fully implemented (does not work in PowerShell). 
https://github.com/mamba-org/mamba/issues/1994#issuecomment-1271462295

mamba-org/mamba#1717

I'm still using a mix of conda and mamba commands.  Perhaps I could move to use only mamba (unless when using PowerShell).

Reply:
We can consider replacing all conda instances with mamba.

At least this one (line 191) should be replaced with mamba when creating the env from yaml:
 `conda env create -n cq-dev -f environment.yml`

Reply:
OK, I did not know that PS is not supported (not using it myself). I agree with the second comment.